README.md

```diff
@@ -0,0 +1,91 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# ToDo List
+
+> "ToDo List" is a simple website that displays a list of activities and allows you to add and remove tasks from that list. In order to achieve this project we rewrote some of our function definitions into class expressions to apply JavaScript best practices and styling the core elements. This app serves the purpse of tracking your activities for the day
+
+
+### ScreenShoot
+![screenshot](./image/Milestone1.png)
+
+
+## Built With
+
+- HTML
+- CSS 
+- JavaScript Classes
+- Linters
+- Vscode
+
+## Live Demo
+
+[Live Demo](https://radiactivejesus.github.io/ToDo-List/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Node and NPM
+- Web browser (Google chrome)
+- Code Editor (VSCode)
+
+### Development (Running locally)
+
+- git clone the project
+
+```bash 
+git clone git@github.com:RadiactiveJesus/ToDo-List.git
+```
+
+- Install Dependencies
+
+```bash
+npm install
+```
```

Comment:
There seem to be some missing instructions in your documentation:
- [ ] I could not bundle (build) your application until I ran **`yarn install`**, kindly add it as a possible command to set up the application for some users.
- [ ] Kindly add instructions on how to bundle the files. ie **`npm run build`**
- [ ] Kindly direct the users on how to run the application locally ie:
  - Open **`dist/index.html`** in live server